components/prospects/ProfileActions.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import IconButton from '@mui/material/IconButton';
+import Router from 'next/router';
+import ConfirmModal from '../helper/ConfirmModal';
+import { red } from '@mui/material/colors';
+
+function ProfileActions({ id, name }) {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const [modalOpen, setModalOpen] = React.useState(false);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  }
+
+  const handleDeleteClick = () => {
+    setModalOpen(true);
+    setAnchorEl(null);
+  }
+
+  const handleDialogClose = () => {
+    setModalOpen(false)
+  }
+
+  function deleteProfile() {
+    Router.push('/prospects/list')
+  }
+
+  return (
+    <>
+      <IconButton
+        sx={{ backgroundColor: "#e2e2e28a" }}
+        onClick={handleClick}
+      >
+        <MoreVertIcon />
+      </IconButton>
+      <Menu
+        id="basic-menu"
+        anchorEl={anchorEl}
+        open={Boolean(anchorEl)}
+        onClose={handleClose}
+        PaperProps={{
+          style: {
+            width: '20ch',
+          },
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'right',
+        }}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'right',
+        }}
+      >
+        <MenuItem onClick={handleDeleteClick}>Delete</MenuItem>
```

Comment:
This button should not be hardcoded as we could use this component in another place where we don't need a "delete" option. So you should get a list of `MenuItem` components as a parameter so we can customize it.